models/FeedbackModel.ts

```diff
@@ -12,8 +13,13 @@ export class FeedbackModel extends BaseEntity {
   @Index('feedback_by_user_index')
```

Comment:
I know this wasn't a new change, but if we're establishing a feedback by user index, there should be some way to query feedback per user, else this is unnecessary write overhead for no read optimizations.

Reply:
Same principle for by event index. @dowhep maybe we should consider an index by event type if we stick to the query params structure, and this can be a consideration for the future.

Reply:
Honestly I feel like rather than/in addition to the query params structure we have a standard GET route for getting all feedback by event uuid, just seems a bit cleaner to me. Having that additional index may be fine with just the query params structure